wordpress-mobile/WordPress-Android
# Update Enhanced Screenshot Translations

Brings in the latest screenshot localizations.

**To Test:**

1. Run `be fastlane update_appstore_strings`. Once the `.po` file has been created, try manually uploading it. You may want to make a small change to one of the source files (`WordPress/metadata/enhanced_screenshot_1.html`, for instance) in order to allow the script to generate an entirely new `.po` file.
2. Run `bundle exec fastlane android download_promo_strings`. Assuming no translators have made changes recently, the repo should be clean – this PR brings in all of the changes that would result from that (as of right now).
